package.json

```diff
@@ -30,7 +30,7 @@
   "scripts": {
     "postinstall": "yarn compile && yarn run readme",
     "test": "yarn workspaces run test",
-    "test:unit": "yarn lerna run test --ignore \"@sap-cloud-sdk/integration-tests\" --ignore \"@sap-cloud-sdk/type-tests\" --ignore \"@sap-cloud-sdk/e2e-tests\" --ignore \"@sap-cloud-sdk/nightly-tests\"",
+    "test:unit": "yarn lerna run test -- --ignore \"@sap-cloud-sdk/integration-tests\" --ignore \"@sap-cloud-sdk/type-tests\" --ignore \"@sap-cloud-sdk/e2e-tests\" --ignore \"@sap-cloud-sdk/nightly-tests\"",
```

Comment:
I guess this is a windows related issue right?

Reply:
Yes, parameter is not forwarded to lerna otherwise, when I add --no-bail in the workflow steps. I checked the behavior when the tests ran on linux, and this didn't have any impact